internal/configs/sharding/testnet.go

```diff
@@ -39,7 +39,7 @@ func (testnetSchedule) BlocksPerEpoch() uint64 {
 }
 
 func (ts testnetSchedule) CalcEpochNumber(blockNum uint64) *big.Int {
-	epoch := blockNum % ts.BlocksPerEpoch()
+	epoch := blockNum / ts.BlocksPerEpoch()
```

Comment:
what about the other networks, is this mistake there too?

Reply:
i checked it’s only a one off mistake and had no real impact before because this method is not really used